Gems/AtomLyIntegration/AtomImGuiTools/Code/Source/AtomImGuiToolsSystemComponent.cpp

```diff
@@ -123,6 +131,21 @@ namespace AtomImGuiTools
             }
             ImGui::EndMenu();
         }
+
+        if (m_switchableRenderPipelines.size() > 0)
+        {
+            if (ImGui::BeginMenu("Render Pipelines"))
+            {
+                for (auto renderPipelinePath : m_switchableRenderPipelines)
```

Comment:
will renderPipelinePath be a copy of a string?
```suggestion
                for (const auto& renderPipelinePath : m_switchableRenderPipelines)
```

Reply:
Good catch. It would be a copy of the string